src/frontend/A32/location_descriptor.h

```diff
@@ -42,6 +45,7 @@ class LocationDescriptor {
         fpscr = (o.Value() >> 32) & FPSCR_MODE_MASK;
         cpsr.IT(ITState{static_cast<u8>(o.Value() >> 40)});
         single_stepping = (o.Value() >> 32) & 4;
+        asid = ((o.Value() >> 32) >> ASID_BIT_SHIFT) & ASID_MASK;
```

Comment:
```suggestion
        asid = static_cast<u8>(((o.Value() >> 32) >> ASID_BIT_SHIFT) & ASID_MASK);
```

Otherwise this truncates implicitly.